10-learning.Rmd

```diff
@@ -19,27 +19,26 @@ help, reading R source code, and use of online material.
-- If stuck, ask for help! A clear question posted in an appropriate place, using reproducible code, should get a quick and enlightening answer.
-- For more in-depth learning, nothing can beat immersive R books and tutorials. Do some research and decide which resources you should complete.
-- One of the best ways to consolidate learning is to write-it-up and pass on the knowledge: telling the story of what you've learned with also help others.
+1. Use R's internal help, e.g. with `?`, `??`, `vignette()` and `appropos()`. Try **swirl**.
+1. Read about the latest developments in established outlets such as the *Journal for Statistical Software*, the *R Journal*, R lists and the 'blogosphere'.
+1. If stuck, ask for help! A clear question posted in an appropriate place, using reproducible code, should get a quick and enlightening answer.
+1. For more in-depth learning, nothing can beat immersive R books and tutorials. Do some research and decide which resources you should complete.
+1. One of the best ways to consolidate learning is to write-it-up and pass on the knowledge: telling the story of what you've learned with also help others.
 
 ## Using R's internal help
 
 The best place to look for help on R functions is often within R itself. We suggest this as the 'first port of call' because it has 3 main advantages from an efficiency perspective, compared with other sources of help. First, it's faster to query R from inside your IDE than to switch context and search for help on a different platform (e.g. the internet which has countless distractions). Second, you can more easily work offline if you learn to master R's internal help. Third, learning to read R's documentation (and source code) is a powerful skill in itself that will improve your R programming.
 
-The main disadvantage of R's internal help is that it is notoriously terse and in some cases sparse. So do not expect to *always* be able to find the answer in R: while we recommend R's internal help is the first place to look, we do not recommend *relying* on it. From a learning perspective becoming aquainted with R's documentation is often better than finding out the solution from a different source: by reading the documentation, you will see what the function was designed to do and perhaps gain further insight about its history, e.g. through references to academic papers.
+The main disadvantage of R's internal help is that it is notoriously terse and in some cases sparse. So do not expect to *always* be able to find the answer in R: while we recommend R's internal help is the first place to look, we do not recommend *relying* on it. From a learning perspective becoming acquainted with R's documentation is often better than finding out the solution from a different source: by reading the documentation, you will see what the function was designed to do and perhaps gain further insight about its history, e.g. through references to academic papers.
```

Comment:
Base R docs is aimed at developers not users. Also, you wouldn't expect the mean function to explain what the mean is.